source/MaterialXRender/OpenGL/GLTextureHandler.cpp

```diff
@@ -67,17 +67,32 @@ bool GLTextureHandler::acquireImage(std::string& fileName,
         switch (imageDesc.channelCount)
         {
         case 3:
-            internalFormat = GL_RGB;
+        {
+            internalFormat = GL_RGBA;
+            // Map {RGB} to {RGB, 1} at shader access time
+            GLint swizzleMasRGB[] = { GL_RED, GL_GREEN, GL_BLUE, GL_ONE };
+            glTexParameteriv(GL_TEXTURE_2D, GL_TEXTURE_SWIZZLE_RGBA, swizzleMasRGB);
```

Comment:
I notice that you have swizzleMaskR, and swizzleMasRG, is that a mistake, ie. the missing 'k'?